### ISSUE TITLE
Missing informaiton for mutualTLS in v3.1.0 Spec

### ISSUE THREAD

**Issue by philsturgeon** [OAI/OpenAPI-Specification#2475]
Just noticed there's no information about mutualTLS in the spec other than that it exists as a valid type.

**Comment by MikeRalphson** [comment]
As intended. See lengthy discussions in #1004 - note specification extensions are allowed also. What fields do you think are missing?

**Comment by philsturgeon** [comment]
I don’t know if it needs any fields but maybe an example or something. I’m just saying I have no idea how it works.

**Comment by MikeRalphson** [comment]
It's just a hint to consoles / SDKs (and all other use-cases) that a client certificate must be obtained **in some way** and presented in the TLS connection.

**Comment by philsturgeon** [comment]
Right, but unless we want to continue the years old OAI trend of making PR comments part of the spec/docs, could we consider making the specs have some information on how mutualTLS works? 😆

**Comment by MikeRalphson** [comment]
> unless we want to continue the years old OAI trend of making PR comments part of the spec/docs

Can you run that sentence by me one more time? I totally don't get it.

**Comment by philsturgeon** [comment]
It was a funny joke about how lots and lots and lots of what we’ve recently had to clarify in the spec (3.0.x) was all “the spec is t clear but Darrel said this in an issue a few years ago” or “this issue says X and that pull request says Y but the spec doesn’t say anything so which is right”.

I’d like to try and get away from that.

Put another way: You’re kind to try and help me understand what mutualTLS might look like, but we could probably put an example in the spec, as we’ve put other security examples in the spec.

**Comment by MikeRalphson** [comment]
> “the spec is clear but Darrel said this in an issue a few years ago” or “this issue says X and that pull request says Y but the spec doesn’t say anything so which is right”. I’d like to try and get away from that.

I see either @Relequestual's proposed use of Architectural Decision Records over at JSON Schema, or the publishing of some "technical notes" as a good way to capture additional information about how the sausage was made, without bloating the spec itself.

I'm still struggling to see what an example would add above the stated support for the type that's already in the spec. But, as ever, PRs for discussion are always welcomed!

**Comment by philsturgeon** [comment]
I'm not looking for anything complex, just consistency. We show samples for other securty schemes so lets add a sample for this one too. https://github.com/OAI/OpenAPI-Specification/pull/2625

**Comment by shriduttkothari** [comment]
I'm also attempting to use the **securityScheme type mutualTLS**. 

But don't know how to use it, thee is not documentation available, and no sample or example avaiable.

Kindly help if anyone knows how to use **securityScheme type mutualTLS**!!

**Comment by LasneF** [comment]
given the subject is quite old , 
given a basic exemple has been provided in the merge request 
given that for mutualTLS there must be direct and long communication between parties, to me the description field is good enough to details what is required

**Comment by lornajane** [comment]
Thanks @LasneF ! Part of the problem here is that the example is in an unreleased spec version, but I think we do have the example requested, so I'll close the issue.
